addons/sale_project_stock/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     'summary': 'Adds a full traceability of inventory operations on the profitability report.',
     'license': 'LGPL-3',
     'category': 'Sales',
-    'depends': ['sale_project', 'sale_stock'],
+    'depends': ['sale_project', 'sale_stock', 'project_stock_account'],
```

Comment:
maybe a migration script is needed to update the dependencies?

Reply:
Do you have the util method in mind ? :innocent:

Reply:
FYI: https://github.com/odoo/upgrade/wiki/Modules-Management#modules-management